ci/purge_expired_questionnaire_state.yaml

```diff
@@ -28,6 +28,8 @@ run:
             datastoreReadGqlQuery="SELECT * FROM \`questionnaire-state\` WHERE submitted_at < $((`date --utc +%s` - EXPIRATION_TIME_OFFSET_IN_SECONDS)) AND submitted_at > NULL",\
             datastoreReadProjectId="${PROJECT_ID}",\
             datastoreDeleteProjectId="${PROJECT_ID}" \
+            --network redis  \
+            --subnetwork regions/europe-west2/subnetworks/redis \
```

Comment:
Happy with this once it is signed off from Mark. Worth mentioning that this `VPC` isn't tied to Redis, it was just named after it as that is the only thing that would use it, but could be renamed to like `eq`.

Also, according to the docs, you don't need the `network` when the `subnetwork` is provided.

https://cloud.google.com/dataflow/docs/guides/specifying-networks
> If you specify a subnetwork, Dataflow chooses the network for you. Therefore, when specifying a subnetwork, you can omit the network parameter.